video/out/drm_common.c

```diff
@@ -627,6 +710,10 @@ void kms_destroy(struct kms *kms)
        drm_atomic_destroy_context(kms->atomic_context);
     }
 
+    if (kms->primary_node_path) {
+        TA_FREEP(&(kms->primary_node_path));
```

Comment:
On second thought I don't think this is needed, talloc attaches the allocation to the `log` context, so it'll get freed eventually.
(cf. `connector_name` which is only manually freed in case of error, [allocated here](https://github.com/mpv-player/mpv/blob/51852aad52080ea8bd6f4b72e63e581dd5234d33/video/out/drm_common.c#L522))